janitor/functions.py

```diff
@@ -1190,9 +1190,23 @@ def change_type(df, column: str, dtype):
     :param df: A pandas dataframe.
     :param column: A column in the dataframe.
     :param dtype: The datatype to convert to. Should be one of the standard
+    :param ignore_exception: The way to handle a conversion exception {False, "FillNaN", "Keep_values"}: default False, "FillNaN": replace non convertible values with None, "Keep_values": keep non convertible values.  
```

Comment:
This might break the pycodestyle checks going forward. Could you ensure that it's kept within 79 characters?